pkg/util/ec2/ec2_test.go

```diff
@@ -107,3 +107,27 @@ func TestExtractClusterName(t *testing.T) {
 		})
 	}
 }
+
+func TestGetNetworkID(t *testing.T) {
```

Comment:
Can we add test for the case where we have no networkID and multiple ones (ie the error cases) ?